packages/aws-cdk-lib/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -295,6 +308,11 @@ export abstract class QueueProcessingServiceBase extends Construct {
    */
   public readonly scalingSteps: ScalingInterval[];
 
+  /**
+   * Grace period after scaling activity.
+   */
```

Comment:
Is there any reason why the additional information in the docs of the other `cooldown` property is not also added here?

Reply:
No reason, added it